app/dashboard/templates/shared/kudos-hex-images.html

```diff
@@ -0,0 +1,107 @@
+          fill="black"
+        ></path>
+        <path
+          d="M55.7216 72.3201V74.0001H48.0736V72.4801L51.9456 68.3041C52.4042 67.8028 52.7402 67.3441 52.9536 66.9281C53.1669 66.5121 53.2736 66.1014 53.2736 65.6961C53.2736 65.1734 53.1189 64.7788 52.8096 64.5121C52.5109 64.2454 52.0789 64.1121 51.5136 64.1121C50.5109 64.1121 49.4816 64.5068 48.4256 65.2961L47.7376 63.7921C48.1856 63.3868 48.7616 63.0668 49.4656 62.8321C50.1696 62.5868 50.8789 62.4641 51.5936 62.4641C52.7349 62.4641 53.6362 62.7468 54.2976 63.3121C54.9696 63.8668 55.3056 64.6188 55.3056 65.5681C55.3056 66.2188 55.1616 66.8321 54.8736 67.4081C54.5962 67.9841 54.1269 68.6294 53.4656 69.3441L50.6656 72.3201H55.7216Z"
+          fill="black"
+        ></path>
+        <path
+          d="M65.3629 72.3201V74.0001H58.3389V72.3201H60.8349V64.9761L58.5149 66.3681V64.5281L61.6829 62.6081H62.8829V72.3201H65.3629Z"
+          fill="black"
+        ></path>
+      </g>
+      <g class="animated-fill">
+        <path
+          fill-rule="evenodd"
+          clip-rule="evenodd"
+          d="M29.9651 1.79458C31.3072 -0.610673 34.7731 -0.594909 36.0933 1.82246L48.3517 24.2691C49.2169 25.8534 48.7353 27.8358 47.2396 28.8465L34.9881 37.1258C33.8146 37.9188 32.2789 37.9264 31.0976 37.1451L18.5696 28.8592C17.0427 27.8492 16.552 25.8332 17.444 24.2345L29.9651 1.79458ZM33.4603 3.26035C33.2718 2.91502 32.7766 2.91276 32.5849 3.25637L20.0638 25.6963C19.9363 25.9247 20.0064 26.2127 20.2246 26.3569L32.7526 34.6429C32.9213 34.7545 33.1407 34.7534 33.3083 34.6401L45.5598 26.3609C45.7735 26.2165 45.8423 25.9333 45.7187 25.707L33.4603 3.26035Z"
+        ></path>
+        <path
+          fill-rule="evenodd"
+          clip-rule="evenodd"
+          d="M33.2769 20.6328C33.1313 20.5529 32.9555 20.5506 32.8079 20.6266L20.0364 27.1957C19.2997 27.5746 18.3953 27.2845 18.0164 26.5479C17.6374 25.8112 17.9275 24.9068 18.6642 24.5279L31.4357 17.9588C32.4693 17.4271 33.6995 17.4433 34.7188 18.002L47.5712 25.0464C48.2977 25.4446 48.5638 26.3563 48.1656 27.0827C47.7675 27.8092 46.8558 28.0753 46.1293 27.6771L33.2769 20.6328Z"
+        ></path>
+        <path
+          fill-rule="evenodd"
+          clip-rule="evenodd"
+          d="M43.6855 34.5207C46.0081 33.2323 48.4718 35.9761 46.9417 38.1471L34.9803 55.1179C34.5336 55.7516 33.8026 56.1236 33.0274 56.1117C32.2521 56.0998 31.5329 55.7055 31.1059 55.0584L19.9596 38.1659C18.4985 35.9515 21.0383 33.2859 23.3208 34.6383L33.0649 40.4121L43.6855 34.5207ZM31.5403 42.9958L23.6596 38.3262L31.5403 50.2697V42.9958ZM34.5403 50.5347V43.0243L43.2321 38.2029L34.5403 50.5347Z"
+        ></path>
+        <path
+          fill-rule="evenodd"
+          clip-rule="evenodd"
+          d="M33.0403 1.36159C33.8688 1.36159 34.5403 2.03316 34.5403 2.86159V36.3076C34.5403 37.136 33.8688 37.8076 33.0403 37.8076C32.2119 37.8076 31.5403 37.136 31.5403 36.3076V2.86159C31.5403 2.03316 32.2119 1.36159 33.0403 1.36159Z"
+        ></path>
+      </g>
+    </g>
+
+  </defs>
+</svg>
```

Comment:
could we keep this as `.svg` file within `images/` +  compress it using `svgo <img-path>` ?

Reply:
@thelostone-mc unfortunately if we want to preserve more or less synced animation or if we want to have it at all - we need to inline those :\
Though we can run them through SVGO to shave off some Kb - around 8-10Kb total. 
What's your call?

Reply:
@thelostone-mc ? :)

Reply:
Hmm then let's run this via svgo and leave it here ?

Reply:
instead of svgo for this situations you can use this amazing tool https://jakearchibald.github.io/svgomg/